WTWSMS/localisation/WtWSMS_religions.csv

```diff
@@ -158,7 +167,9 @@ twelver_DESC;Twelver Shia or the Ithna'ashariyyah' is the largest branch of Shia
 ismaili;Ismaili;;;;;;;;;;;x
 ismaili_DESC;Ismailis get their name from their acceptance of Isma'il ibn Jafar as the divinely appointed spiritual successor to Ja'far al-Sadiq, wherein they differ from the Twelvers, who accept Musa al-Kadhim, younger brother of Isma'il, as the true Imam. Over time, the teachings of Ismailism further transformed into the belief system as it is known today, with an explicit concentration on the deeper, esoteric meaning (ba?in) of the faith. With the eventual development of Twelverism into the more literalistic (zahir) oriented Akhbari and later Usuli schools of thought, Shiaism developed in two separate directions: the metaphorical Ismailli group focusing on the mystical path and nature of God and the divine manifestation in the personage of the "Imam of the Time" as the "Face of God", with the more literalistic Twelver group focusing on divine law and the deeds and sayings Muhammad and his successors, who as A'immah were guides and a light to God.;;;;;;;;;;;x
 wahhabi;Wahhabi;;;;;;;;;;;x
-wahhabi_DESC;Wahhabism is an ultra-conservative branch or sect of Sunni Islam. It is a religious movement among fundamentalist Islamic believers, with an aspiration to return to the earliest fundamental Islamic sources of the Quran and Hadith, with inspiration from the teachings of Medieval theologian Ibn Taymiyyah and early jurist Ahmad ibn Hanbal.;;;;;;;;;;;x
+wahhabi_DESC;Wahhabism is an ultra-conservative, extremist, violent branch or sect of Sunni Islam. It is a religious movement among fundamentalist Islamic believers, with an aspiration to return to the earliest fundamental Islamic sources of the Quran and Hadith, with inspiration from the teachings of Medieval theologian Ibn Taymiyyah and early jurist Ahmad ibn Hanbal. Wahhabis refuse to tolerate the presence of any other religion and seek to spread their faith by the sword. Wahhabism would remain a violent force of Islam for centuries after many other Islamic sects died, going toe to toe with the greatest empires and countries the world has ever known, and successfully blindsiding them with sporadic acts of terror and violence.;;;;;;;;;;;x
+wahhabi_adj;Wahhabi;;;;;;;;;;;x
+wahhabi_FEATURES;�Y*�! Wahabbis are extremely aggressive and zealous \n�Y*�! Wahhabis have exceptional fighting abilities \n�Y*�! Wahhabis experience boosts in morale and army tactics \n�Y*�! Wahhabis have an attrition advantage in their homelands due to their scorched-earth tactics and use of unconventional warfare;;;;;;;;;;;x
```

Comment:
Wahhabism in the mod doesn't fit with that description at start, so I think we will have to revert those changes.